packages/server/package.json

```diff
@@ -1,3 +1,10 @@
 {
-  "name": "@course-tracker/server"
+  "name": "@course-tracker/server",
+  "dependencies": {
+    "bcrypt": "^5.0.1",
+    "express": "^4.17.1",
+    "express-session": "^1.17.2",
+    "mongoose": "^5.13.8",
+    "sessions": "0.0.2-7"
+  }
```

Comment:
This should probably add a dev script using `nodemon` and a start script using `node`.